### ISSUE TITLE
Speak mode causing crash on Mac

### ISSUE THREAD

**Issue by hartsteins1** [Significant-Gravitas/Auto-GPT#4923]
I'm having repeated crashes any time trying to access Speak mode.  I have installed auto-get through every method described (Docker, git....etc) and on multiple Macs. The same errors appears and auto-gpt crashes.  I am trying to use ElevenLabs voices and have included their API Keys and voice ID in the .env file.

Ideas?

Thank you
-----------------------

Speak Mode:  ENABLED

....... (removed NEWS FEED).....

Traceback (most recent call last):
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "/Users/karen/Documents/Auto-GPT/autogpt/__main__.py", line 5, in <module>
    autogpt.cli.main()
  File "/Users/karen/Documents/Auto-GPT/venvAutoGPT/lib/python3.11/site-packages/click/core.py", line 1157, in __call__
    return self.main(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/karen/Documents/Auto-GPT/venvAutoGPT/lib/python3.11/site-packages/click/core.py", line 1078, in main
    rv = self.invoke(ctx)
         ^^^^^^^^^^^^^^^^
  File "/Users/karen/Documents/Auto-GPT/venvAutoGPT/lib/python3.11/site-packages/click/core.py", line 1666, in invoke
    rv = super().invoke(ctx)
         ^^^^^^^^^^^^^^^^^^^
  File "/Users/karen/Documents/Auto-GPT/venvAutoGPT/lib/python3.11/site-packages/click/core.py", line 1434, in invoke
    return ctx.invoke(self.callback, **ctx.params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/karen/Documents/Auto-GPT/venvAutoGPT/lib/python3.11/site-packages/click/core.py", line 783, in invoke
    return __callback(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/karen/Documents/Auto-GPT/venvAutoGPT/lib/python3.11/site-packages/click/decorators.py", line 34, in new_func
    return f(get_current_context(), *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/karen/Documents/Auto-GPT/autogpt/cli.py", line 96, in main
    run_auto_gpt(
  File "/Users/karen/Documents/Auto-GPT/autogpt/main.py", line 157, in run_auto_gpt
    ai_config = construct_main_ai_config(config)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/karen/Documents/Auto-GPT/autogpt/prompts/prompt.py", line 62, in construct_main_ai_config
    logger.typewriter_log(
  File "/Users/karen/Documents/Auto-GPT/autogpt/logs.py", line 92, in typewriter_log
    say_text(f"{title}. {content}")
TypeError: say_text() missing 1 required positional argument: 'config'
Press any key to continue...

**Comment by pineappleslices** [comment]
I'm having the same issue for windows. I've modified the logs.py/elevenlabs.py/base.py/etc file to handle the issues of " missing config, attrib, etc". I basically walked up the errors thrown and slowly adjusted them with small tweaks. Some errors where very challenging as they were coming from the sit package as one or more issues. So, after many many attempts I've managed to get speak mode enabled, and run autogpt, but I'm stuck chasing errors in my prompt which basically makes the commands unreadable. I'm going to continue chasing errors until we get a fix from devs :P
